clients/kafka-common/src/main/java/org/eclipse/hono/client/kafka/AbstractKafkaConfigProperties.java

```diff
@@ -106,13 +101,30 @@ public final void overrideComponentUidUsedForClientId(final String componentUId)
     /**
      * Checks if a configuration has been set.
      *
-     * @return {@code true} if the {@value #PROPERTY_BOOTSTRAP_SERVERS} property has been configured with a non-null
-     *         value.
+     * @return {@code true} if the {@value CommonClientConfigs#BOOTSTRAP_SERVERS_CONFIG} property has been configured
```

Comment:
When I try to create JavaDoc on my machine as part of the Maven build, this breaks the build because the JavaDoc generator cannot resolve the reference. Does this work on your machine?

Reply:
My bad, I had only added the `CommonClientConfigs` but did not adapt the constant name ... seems to work.